docs/quickstart.zh.md

```diff
@@ -20,6 +22,22 @@ sh -c "$(curl -fsSL https://raw.githubusercontent.com/devstream-io/devstream/mai
 
 > 可选：建议你将 dtm 移动到包含于 PATH 的目录下，比如 `mv dtm /usr/local/bin/`。
 
+### 用 [asdf](https://asdf-vm.com/) 安装
+
+```shell
+# Show all installable versions
+asdf list-all dtm
+
+# Install specific version
+asdf install dtm latest
+
+# Set a version globally (on your ~/.tool-versions file)
+asdf global dtm latest
+
+# Now dtm commands are available
+dtm --help
+```
```

Comment:
```shell
# asdf list-all dtm
No such plugin: dtm
```
Please resolve this problem, thank you! ❤️

Reply:
I have forgot  the description for adding asdf-dtm plugin. Installing asdf is simple. I think it is ok to put asdf link directly.